### PR TITLE
feat: updated token safety copy

### PR DESCRIPTION
**Pull request by cartcrom**

### CODE REVIEW

File: src/constants/tokenSafety.tsx [hunk 1]
```diff
@@ -14,17 +14,36 @@ export function getWarningCopy(warning: Warning | null, plural = false) {
...
-        <Plural
-          value={plural ? 2 : 1}
-          _1="You can't trade this token using the Uniswap App."
-          other="You can't trade these tokens using the Uniswap App."
-        />
-      )
+    switch (warning.level) {
+      case WARNING_LEVEL.MEDIUM:
+        heading = (
+          <Plural
```
Comment (vm) [inline src/constants/tokenSafety.tsx:20]: if you have `plural`, why do you need to pass it into this component? can't you just do `plural ? text1 : text2?`

**Review by vm — APPROVED** [review]

File: src/constants/tokenSafety.tsx (hunk 1 above)
Comment (cartcrom) [reply src/constants/tokenSafety.tsx:20]: Plural is a replacement for using ternary \<Trans> logic

**Comment by cartcrom** [comment]
will merge this branch in tandem with some other safety changes after approval from legal

File: src/constants/tokenSafety.tsx (hunk 1 above)
Comment (vm) [reply src/constants/tokenSafety.tsx:20]: ah

**Pull request closed without merging**